### PR TITLE
Release 1.6.0.

### PR DESCRIPTION
**Pull request by davidmorgan**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,25 @@
...
+- Enforce six new lint rules:
+
+  - [`curly_braces_in_flow_control_structures`]
+  - [`empty_catches`]
+  - [`library_names`]
+  - [`library_prefixes`]
+  - [`type_init_formals`]
+  - [`unnecessary_null_in_if_null_operators`]
+
+[`curly_braces_in_flow_control_structures`]: http://dart-lang.github.io/linter/lints/curly_braces_in_flow_control_structures.html
```
Comment (kevmoo) [inline CHANGELOG.md]: this should be `https` URLs

Comment (davidmorgan) [reply CHANGELOG.md]: Done, thanks!

**Comment by kevmoo** [comment]
Now we see how many packages w/ `^1.0.0` start breaking on travis with `--fatal-infos` 😄

**Comment by davidmorgan** [comment]
Whoops, of course an important part is to update the version number in pubspec.yaml. Will do that too...

**Pull request merged**